services/ui-src/src/components/fields/NumberField.test.tsx

```diff
@@ -283,7 +283,7 @@ describe("<NumberField />", () => {
         "[name='testNumberField']"
       )!;
       const displayValue = numberField.value;
-      expect(displayValue).toEqual("54,321");
+      expect(displayValue).toEqual("54321");
```

Comment:
tests were expecting a comma where no mask was applied. fix test to check for the actual expected output